### PR TITLE
Fix view of nested array field in table column

### PR DESCRIPTION
**Pull request by z0h3**
**What this PR does / why we need it**:
Nested complex items of arrays views as `[object Object]` string.
image

**Which issue(s) this PR fixes**:
This PR improves viewing of all arrays by serialization them to json string.
image

### CODE REVIEW

File: src/dependencies/flatten.ts [hunk 1]
```diff
@@ -12,7 +12,9 @@ export default function flatten(target: object, opts?: { delimiter?: any; maxDep
   function step(object: any, prev: string | null) {
     Object.keys(object).forEach(key => {
       const value = object[key];
-      const isarray = opts?.safe && Array.isArray(value);
+      const isSafe = opts?.safe;
+      const isarray = Array.isArray(value);
```
Comment (idastambuk) [inline src/dependencies/flatten.ts:16]:
```suggestion
      const isArray = Array.isArray(value);
```

File: src/dependencies/flatten.ts [hunk 2]
```diff
@@ -12,7 +12,9 @@ export default function flatten(target: object, opts?: { delimiter?: any; maxDep
...
+      const isSafeArray = isSafe && isarray;
```
Comment (idastambuk) [inline src/dependencies/flatten.ts:17]:
```suggestion
      const isSafeArray = isSafe && isArray;
```

File: src/dependencies/flatten.ts [hunk 3]
```diff
@@ -12,7 +12,9 @@ export default function flatten(target: object, opts?: { delimiter?: any; maxDep
...
       const type = Object.prototype.toString.call(value);
       const isobject = type === '[object Object]';
```
Comment (idastambuk) [inline src/dependencies/flatten.ts:19]:
```suggestion
      const isObject = type === '[object Object]';
```

File: src/dependencies/flatten.ts [hunk 4]
```diff
@@ -22,12 +24,16 @@ export default function flatten(target: object, opts?: { delimiter?: any; maxDep
         maxDepth = currentDepth + 1;
       }
 
-      if (!isarray && isobject && Object.keys(value).length && currentDepth < maxDepth) {
+      if (!isSafeArray && isobject && Object.keys(value).length && currentDepth < maxDepth) {
```
Comment (idastambuk) [inline src/dependencies/flatten.ts:27]:
```suggestion
      if (!isSafeArray && isObject && Object.keys(value).length && currentDepth < maxDepth) {
```

**Review by idastambuk — APPROVED** [review]

**Comment by idastambuk** [comment]
Hello @z0h3, thanks for this contribution! I've just added some nitpicks as capitalization suggestions :)

**Comment by z0h3** [comment]
@iwysiu Hi!
Can you review this please?

**Review by iwysiu — APPROVED** [review]
Hi @z0h3 ! Sorry for the delay. Thanks for the contribution, it looks good!

**Pull request merged**